Blueprints/BlueprintDefinitions/vs2022/EmptyServerless/template/src/BlueprintBaseName.1/Function.cs

```diff
@@ -1,6 +1,8 @@
 using System.Net;
 using Amazon.Lambda.Core;
 using Amazon.Lambda.APIGatewayEvents;
+using Amazon.Lambda.Annotations;
```

Comment:
All my comments for the image based blueprint also apply to the non-image based template.

Reply:
Done.